src/js/components/Form/stories/DynamicFields.js

```diff
@@ -16,7 +16,7 @@ export const DynamicFields = () => {
           onSubmit={({ value }) => console.log('Submit', value)}
         >
           <FormField label="Name" name="name" required>
-            <TextInput name="name" />
+            <TextInput name="name" aria-label="name" />
```

Comment:
Also, if this is the standard direction to head, should TextInput automatically set `aria-label` to `name` if `aria-label` isn't set otherwise?

Reply:
https://github.com/grommet/grommet/pull/6466#discussion_r1015940492

>I think there is a trade-off for that.
Setting it by default will be good for the average user that forgets to pay attention to aria and accessibility attributes. BUT, for the more savvy accessibility users that would like to use `aria-describedby` or `aria-description` instead of `aria-label`, setting a default of aria-label will clutter and overuse the screen reader content which IMHO is unfair.